src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1314,4 +1314,24 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="NoPackageSelectionArgumentProvided" xml:space="preserve">
     <value>No package selection argument was provided; see the help for details about finding a package.</value>
   </data>
+  <data name="PortableInstallFailed" xml:space="preserve">
+    <value>Portable install failed; Cleaning up...</value>
+  </data>
+  <data name="BothPurgeAndPreserveFlagsProvided" xml:space="preserve">
+    <value>Both `purge` and `preserve` args are provided</value>

```

Comment:
```suggestion
    <value>Both `purge` and `preserve` arguments are provided</value>
```
Don't abbreviate as it can make it harder for non-English speakers to machine-translate for languages we don't have localizations for.

Reply:
I actually followed the pattern of an existing string, so I updated both to use `arguments` instead of `args`